doc/CodeDocumentation.conf.in

```diff
@@ -1125,7 +1125,13 @@ IMAGE_PATH             =
 # need to set EXTENSION_MAPPING for the extension otherwise the files are not
 # properly processed by doxygen.
 
-INPUT_FILTER           =
+# Allow the use of standard LaTeX formulas in MFEM comments by replacing
+# $$...$$ with \f[...\f] and $...$ with \f$...\f$ before running Doxygen.
```

Comment:
Could we have it replace `\[` and `\]` with `\f[` and `\f]` also?

Reply:
Certainly

Reply:
If we want to add this, the regex pattern will be `s/\\\[((?:.|\n)+?)\\\]/\\f[\1\\f]/g;`